addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -888,6 +888,10 @@ body.editor_enable.editor_has_snippets {
             }
         }
 
+        we-select.o_we_large_select > div {
```

Comment:
I totally agree with you
That's what I expected to see.

So I refactor all the code that uses we_large_input
Maybe it would be better to make a separate task for that.
If you agree I can create it and maybe do it on Friday before the end of my internship :wink:

Reply:
There's really no need to create a new task just for that :wink: It's a need which comes up with this task, at worst it can be another commit in this PR. Here I am only asking about renaming two classes ^^